frontend/src/hooks/tripPlans.ts

```
@@ -8,4 +8,21 @@ export const fetchTripPlans = async (): Promise<TripPlan[]> => {
+        const response = await axios.post<RegisterTripPlanResponse>('http://localhost/api/trip-plan', tripPlan)
+        return response.data.id
+    } catch (error: any) {
+        throw error;
+    }
+};
+
+export const uploadImage = async (tripID: string) => {
+    try {
+        await axios.post(`http://localhost/api/trip-plan/${tripID}/image`);
+    } catch (error: any) {
+        throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)